Xamarin.Forms.Platform.iOS/ExportRendererAttribute.cs

```diff
@@ -1,16 +1,19 @@
 using System;
+using Xamarin.Forms.Core;
 
 #if __MOBILE__
 using UIKit;
+#else
+using Foundation;
 #endif
 
 namespace Xamarin.Forms
 {
 	[AttributeUsage(AttributeTargets.Assembly, AllowMultiple = true)]
-	public sealed class ExportRendererAttribute : HandlerAttribute
+	public sealed class ExportRendererAttribute : BaseExportRendererAttribute
 	{
 #if __MOBILE__
-		public ExportRendererAttribute(Type handler, Type target, UIUserInterfaceIdiom idiom) : base(handler, target)
+		public ExportRendererAttribute(Type handler, Type target, UIUserInterfaceIdiom idiom) : this(handler, target)
```

Comment:
Call the other constructor to avoid writing the same line of MajorVersion code twice.